juriscraper/opinions/united_states/state/nyappterm_1st.py

```diff
@@ -83,3 +108,39 @@ def extract_from_text(self, scraped_text: str) -> Dict[str, Any]:
             },
         }
         return metadata
+
+    def _download_backwards(self, dates: Tuple[date]) -> None:
+        """Make custom date range request
+
+        :param dates: (start_date, end_date) tuple
+        :return None
+        """
+        logger.info("Backscraping for range %s %s", *dates)
+        self._set_parameters(*dates)
+        self.html = self._download()
+        self._process_html()
+
+    def make_backscrape_iterable(self, kwargs: dict) -> None:
+        """Checks if backscrape start and end arguments have been passed
+        by caller, and parses them accordingly
+
+        :param kwargs: passed when initializing the scraper, may or
+            may not contain backscrape controlling arguments
+
+        :return None
+        """
+        start = kwargs.get("backscrape_start")
+        end = kwargs.get("backscrape_end")
+
+        if start:
+            start = datetime.strptime(start, "%m/%d/%Y")
+        else:
+            start = self.first_opinion_date
```

Comment:
I think this `start = self.first_opinion_date` will be overwritten in _set_parameters() if you/ don't run the backscraper with and dont pass an end_date, It is expected to work like this so that it does not start from 08/25/2003?